src/test/java/collectionsEducation/CountryWorker.java

```diff
@@ -4,88 +4,189 @@
 
 public class CountryWorker {
 
+
     public static void main(String[] args) {
         List<String> names = TestDataBuilder.getCountriesNamesList();
         List<CountryDTO> allCountries = TestDataBuilder.getCountriesList();
 //        тут будеш визивать потрібний метот
+        System.out.print("LIST COUNTRY :");
+
+        for (String test : names) {
+                System.out.print(test + " ,");
+        }
+        System.out.println('\n');
+        System.out.println("Задача1 :" + getListOfNamesEquals(names,"United Arab Emirates"));
+
         System.out.println();
-    }
+        System.out.println("Задача2 :" + getNameEquals(names,"GerManY"));
+
+        System.out.println();
+        System.out.println("Задача3 :" + getSortListOfNames(names,false));
+
+        System.out.println();
+        System.out.println("Задача4 :" + getListOfCountriesNameEquals(allCountries,"iTaLy"));
+
+        System.out.println();
+        System.out.println("Задача5 :" + getListOfCountriesNameContains(allCountries,"uniTed"));
+
+        System.out.println();
+        System.out.println("Задача6 :" + getListOfCountriesByName(allCountries,"united",false));
+
+        System.out.println();
+        System.out.println("Задача7 :" + getListOfCountriesByLanguage(allCountries,"fra"));
 
-    private List<String> getListOfNamesEquals(List<String> names, String name) {
+        System.out.println();
+        System.out.println("Задача8 :" + getListOfCountriesByLanguageMap(allCountries,"eng"));
+
+        System.out.println();
+        System.out.println("Задача9 :" + getListOfCountriesByLanguageMap(allCountries));
+
+
+
+
+
+    }
+    private static List<String> getListOfNamesEquals(List<String> names, String name) {
         List<String> filteredCountries = new ArrayList<>();
-//    Замість мого коментаря повинна бути реалізація
+
+        for (String country : names) {
+            if(country.equalsIgnoreCase(name)){
```

Comment:
Apply formatting for all file: 
select all text in class - cntrl(cmnd)+A and then cntl(cmd)+option+L

It will apply standard formatting of java file for more readable